src/views/homepage/components/featured-content-grid/featured-content-grid.module.css

```diff
@@ -11,8 +11,15 @@
 		'b a a'
 		'b c d';
 	grid-template-columns: repeat(3, 1fr);
-	margin-bottom: 237px;
-	margin-top: 60px;
+	margin-bottom: 188px;
+	margin-top: 52px;
+
+	/* 1000px with 16px base font */
+	@media screen and (max-width: 62.5rem) {
+		display: flex;
+		flex-direction: column;
+		gap: 40px;
+	}
```

Comment:
🗒️ I found that these `@media` queries based on `rem` units help us be more responsive to text-only zoom!